roles/postgres/tasks/main.yml

```diff
@@ -42,28 +42,17 @@
     enabled: true
   become: true
 
-- name: Setup postgresql databases
```

Comment:
This has to be executed after we create a user, or otherwise setting a database owner will fail due to non-existent role.